### ISSUE TITLE
code reproduction issue

### ISSUE THREAD

**Issue by DIY-Z** [SCLBD/DeepfakeBench#31]
By training on the source code you provided on UCF, training on FF++c23, and testing on FF++c40, the test results are as follows:
|         |       AUC       |      Acc       |
| :-----: | :-------------: | :------------: |
| FF++c23 | AUC(0.98335639) | Acc(0.9486571) |
| FF++c40 | AUC(0.78450657) | Acc(0.6306857) |

I've noticed that the results do not match what is listed in the paper. Could you please help me understand the possible reasons for this discrepancy? Is it related to data processing? I didn't use the dataset extraction code mentioned in the paper but instead utilized pre-extracted video frames, with 270 frames extracted per video in the training set and 100 frames in the test set.

Is it a matter of model weight selection? When testing, I chose the weights based on the best AUC or ACC on the FF++c23 validation set. May I ask what criteria you used to select model weights during testing?

**Comment by lamb-wolf1** [comment]
The data I got is similar to yours

**Comment by YZY-stack** [comment]
I use only 32 frames for training and testing in the DeepfakeBench paper. This alignment can be critical, I guess.

**Comment by DIY-Z** [comment]
Yeah,when I extract 32 frames from each video, I can indeed achieve results close to the paper. Thanks a lot.
